_resources/port1.0/group/python-1.0.tcl

```diff
@@ -577,5 +586,8 @@ pre-test {
         if {$libdirs ne ""} {
             test.env-append PYTHONPATH=[join $libdirs :]
         }
+        if {${python.pep517} && [option python.pep517_backend] eq "maturin"} {
+            test.env-append PYTHONPATH=${destroot}${frameworks_dir}/Python.framework/Versions/${python.branch}/lib/python${python.branch}/site-packages
```

Comment:
@reneeotten I think that this can be a default value for `PYTHONPATH` when it isn't specified.

`test` is running after `destroot`, and use target `site-packages` as library for package seems quite logical.

Also, I feel that it allows to removes near of all possible cases when `PYTHONPATH` is required.

That do you think?